docker-compose.yml

```diff
@@ -50,29 +47,8 @@ services:
       interval: 15s
       timeout: 10s
       retries: 3
-  worker-web:
```

Comment:
Why don't we need `worker-web` anymore?

Reply:
During development environment, we do not need worker-web (suggested by Harshad) and it takes up a huge amount of time for setting up the docker environment. That is the reason it was decided to remove it